Update add-password-reset-policy.md (#77362)
Azure/azure-content: articles/active-directory-b2c/add-password-reset-policy.md

1. From the sign-up and sign-in page, the user clicks the **Forgot your password?** link. Azure AD B2C initiates the password reset flow.

> 2. The user provides their email address and selects **Send verification code**. Azure AD B2C will then send the user a verification code.

Reviewer: ```suggestion
2. The user provides their email address and selects **Send verification code**. Azure AD B2C sends the verification code to the user's inbox. The user copies the verification code from the email, enters the code in the Azure AD B2C password reset page, and selects **Verify code**.
```

3. The user can then enter a new password.

* The user needs to open the mail box and copy the verification code. The user then enters the verification code in Azure AD B2C password reset page, and selects **Verify code**.

> > [!NOTE]

Reviewer: remove this "NOTE":

```suggestion

```

> > After the email is verified, the user can still select **Change e-mail**, type the other email, and repeat the email verification from the beginning.

Reviewer: remove the line here and we'll incorporate the info into the new text:
```suggestion

```

> > [!TIP]

Reviewer: See the next comment -- I think this works well as a "NOTE"
```suggestion
> [!NOTE]
```

> > In case you would prefer to hide the "Change Email" button on the screen, it can be achieved by hiding the respective HTML element(s) using CSS manipulation . 

Reviewer: I have some suggested edits to this new text. Also, I think we can change the "TIP" to a "NOTE" and reformat lines 38 through 52 as follows:

```suggestion
> After the email is verified, the user can still select **Change email**, type the another email, and repeat the email verification from the beginning. If you'd prefer to hide the **Change email** button, you can modify the CSS to hide the associated HTML element(s) on the page. For example, add the CSS entry below to the selfAsserted.HTML and [customize the user interface with HTML templates](customize-ui-with-html?pivots=b2c-user-flow):
>
> ```html
> <style type="text/css">
>    .changeClaims
>    {
>      visibility: hidden;
>    }
> </style>
> ```
>
> Note that the default name of the **Change email** button in the selfasserted.html page is `changeclaims`. You can find this name by inspecting the page source of the sign-up page using a browser tool such as Inspect.
```
